src/Rubric/RubScrolledTextMorph.class.st

```diff
@@ -808,11 +811,11 @@ RubScrolledTextMorph >> mouseDown: event [
 ]
 
 { #category : #'event handling' }
-RubScrolledTextMorph >> mouseWheel: event [ 
+RubScrolledTextMorph >> mouseWheel: event [
```

Comment:
Is this method ever called?

Reply:
I'm not sure. 

For Linux the mouse wheel also works in a code pane with much text (even without this fix).